From 79c3454d0de816ce0907a29da4de805730235f72 Mon Sep 17 00:00:00 2001
From: Edouard CHIN <chin.edouard@gmail.com>
Date: Fri, 15 Apr 2022 12:20:31 +0200
Subject: [PATCH] File added

---
 blabla.rb | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 blabla.rb

diff --git a/blabla.rb b/blabla.rb
new file mode 100644
index 0000000..d3dfbe4
--- /dev/null
+++ b/blabla.rb
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class FileAdded
+  def test1
+  end
+
+  def test2
+    "This file is just added"
+  end
+end
