From 83f1c013cf33276439ee6577f5bf9c9307f294a7 Mon Sep 17 00:00:00 2001
From: Edouard CHIN <chin.edouard@gmail.com>
Date: Fri, 15 Apr 2022 12:23:10 +0200
Subject: [PATCH] Line removed

---
 app/my_file.rb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/app/my_file.rb b/app/my_file.rb
index 03d99f2..d3dfbe4 100644
--- a/app/my_file.rb
+++ b/app/my_file.rb
@@ -6,6 +6,5 @@ def test1
 
   def test2
     "This file is just added"
-    "This is a new line"
   end
 end
